win_build_output/midl/google_update/x64/google_update_idl.h

```diff
@@ -0,0 +1,5747 @@
+
+
+/* this ALWAYS GENERATED file contains the definitions for the interfaces */

```

Comment:
how is this file generated?

Reply:
During the build, `midl.py` generates these file in temp directory and compares it with pre-installed version.

Reply:
then why are we copying the output? Shouldn't we be modifying the inputs so we get the correct output?

Reply:
https://github.com/brave/brave-browser/pull/303 handles this.